src/main/java/stringcalculator/OperandParser.java

```diff
@@ -0,0 +1,35 @@
+package stringcalculator;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+public class OperandParser {
+	private static final String DEFAULT_INPUT_REGEX = "//(.)\n(.*)";
+	private static final String DEFAULT_OPERAND_DELIMITER_REGEX = ",|:";
+
+	public List<Operand> parse(String input) {
+		if (input == null || input.isBlank()) {
+			return List.of(new Operand("0"));
+		}
+
+		return Stream.of(extractTokens(input))
+			.map(Operand::new)
+			.toList();
+	}
+
+	private static String[] extractTokens(final String input) {
+		Matcher m = Pattern.compile(DEFAULT_INPUT_REGEX).matcher(input);
+		String[] tokens;
+
+		if (m.find()) {
+			String customDelimiter = m.group(1);
+			tokens = m.group(2).split(Pattern.quote(customDelimiter));
+		} else {
+			tokens = input.split(DEFAULT_OPERAND_DELIMITER_REGEX);
+		}
+
+		return tokens;
```

Comment:
가급적 else를 사용하지 않고 [early return](https://www.youtube.com/shorts/UB0Jy6joK-8)을 활용해보면 어떨까요?  

```suggestion
		if (m.find()) {
			String customDelimiter = m.group(1);
			return m.group(2).split(Pattern.quote(customDelimiter));
		}
			
	         return input.split(DEFAULT_OPERAND_DELIMITER_REGEX);
```

Reply:
반영해 보겠습니다.